phpunit/class-gutenberg-rest-global-styles-revisions-controller-test.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+class Gutenberg_REST_Global_Styles_Revisions_Controller_Test extends WP_Test_REST_Controller_Testcase {
+	/**
+	 * @var int
+	 */
+	protected static $admin_id;
+
+	/**
+	 * @var int
+	 */
+	protected static $global_styles_id;
+
+	public function set_up() {
+		parent::set_up();
+		switch_theme( 'emptytheme' );
+	}
+
+	/**
+	 * Create fake data before our tests run.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetupBeforeClass( $factory ) {
+		self::$admin_id = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+		// This creates the global styles for the current theme.
+		self::$global_styles_id = wp_insert_post(
+			array(
+				'post_content' => '{"version": ' . WP_Theme_JSON_Gutenberg::LATEST_SCHEMA . ', "isGlobalStylesUserThemeJSON": true }',
+				'post_status'  => 'publish',
+				'post_title'   => __( 'Custom Styles', 'default' ),
+				'post_type'    => 'wp_global_styles',
+				'post_name'    => 'wp-global-styles-emptytheme',
+				'tax_input'    => array(
+					'wp_theme' => 'emptytheme',
+				),
+			),
+			true
+		);
+	}
+
+	/**
+	 * @covers Gutenberg_REST_Global_Styles_Revisions_Controller::register_routes
+	 */
+	public function test_register_routes() {
+		$routes = rest_get_server()->get_routes();
+		$this->assertArrayHasKey(
+			// '/wp/v2/global-styles/(?P<parent>[\d]+)/revisions',
```

Comment:
Is this a stray comment? It looks identical to the following line 😄

Reply:
🤦